crates/types/src/rpc/transactions.rs

```diff
@@ -313,11 +374,14 @@ pub struct FunctionInvocation {
     messages: Vec<MessageToL1>,
```

Comment:
Should OrderedEvent and OrderedMessageToL1

Reply:
[6e3d672](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/230/commits/6e3d672a5b0fdfc6a9b24793ebd03b989552cfeb)